pkg/apis/kubeone/v1beta2/conversion.go

```diff
@@ -84,3 +84,22 @@ func Convert_kubeone_Features_To_v1beta2_Features(in *kubeoneapi.Features, out *
 func Convert_v1beta2_Features_To_kubeone_Features(in *Features, out *kubeoneapi.Features, s conversion.Scope) error {
 	return autoConvert_v1beta2_Features_To_kubeone_Features(in, out, s)
 }
+
+// Convert_v1beta2_CiliumSpec_To_kubeone_CiliumSpec is an autogenerated conversion function.
```

Comment:
It's not autogenerated. :) And to be fair, I don't think we even need that comment.
```suggestion

```